alexandria/core/serializers.py

```diff
@@ -324,3 +311,20 @@ class Meta:
             "content",
         )
         extra_kwargs = {"content": {"write_only": True}}
+
+
+class WebDAVSerializer(BaseSerializer):
+    webdav_url = serializers.SerializerMethodField()
+
+    def get_webdav_url(self, instance):
+        if not settings.ALEXANDRIA_USE_MANABI:
+            return None
+
+        request = self.context.get("request")
+        host = request.get_host() if request else "localhost"
+        user, group = self.get_user_and_group()
+        return instance.get_latest_original().get_webdav_url(user, group, host)
```

Comment:
Where do we check the permission?

Reply:
This should be done in the `ViewSet`. It inherits from `PermissionViewMixin`. But I guess this doesn't work like that, because we're making `GET` requests and are not creating or updating any resources in the POV of the rest api.

Reply:
Permissions are checked with DGAP, with the Permisison and Visibility view mixins

Reply:
`check_permissions` and `check_object_permissions` in the WebDAV view are being used to call DGAP on `get` request.